### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by sushruth** [bbenoist/vscode-shell#2]
Please do not show the ribbon on top for every shell action if the output window is already visible.

**Comment by bbenoist** [comment]
Hi. Thanks for the suggestion!

By ribbon, do you mean the error message which gets displayed once a command returns a non-zero exit code?

![image](https://cloud.githubusercontent.com/assets/2091902/15375762/5be4fa32-1d51-11e6-86f2-39b5f40a5fda.png)

If yes, I'm afraid the vscode API lacks the information about whether the command output is already opened or not. But I will try to find a solution for this.

One alternative would be to use status bar instead of the regular error dialog but I'm afraid people would end-up not remarking it...

Here is an overview of what it would look like:

![image](https://cloud.githubusercontent.com/assets/2091902/15376031/e23bbef8-1d52-11e6-8c62-761689425fb9.png)

**Comment by sushruth** [comment]
Yes. That is what I was referring to. Perhaps the second idea is the better one. The only state where you would absolutely need that ribbon on top is when the plugin presents "terminate script" option. Other than that, I think the second option is sufficient, in my opinion.
